src/main/java/com/school/schedule/app/Parent.java

```diff
@@ -0,0 +1,39 @@
+package com.school.schedule.app;
+
+import java.util.List;
+
+public class Parent extends Person{
```

Comment:
`Parent` doesn't need `schedule` and `school` variables associated with it which are included in the `Person` class. It's possible that a Parent has multiple children in different schools meaning they shouldn't be associated to just one. Consider moving the `schedule` and `school` variables to both the `Student` and `teacher` classes instead.

Reply:
That makes a lot of sense. I made the changes.